src/App.js

```diff
@@ -8,35 +8,38 @@ import Footer from "./components/common/Footer";
 import Home from "./components/home/Home";
 import StaffList from "./components/staff/StaffList";
 import PetsList from "./components/pets/PetsList";
-
+import { Switch, Route, Redirect } from "react-router-dom";
 /*
   Data
   ---------------
   Note: Normally this data would be pulled from an API. It is not necessary, however, for this application.
 */
-import { employees } from "./data/employees.js";
-import { owners } from "./data/owners";
-import { pets } from "./data/pets";
+import employees from "./data/employees.js";
+import owners from "./data/owners";
+import pets from "./data/pets";
 
 class App extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      employees,
-      owners,
-      pets,
-    };
-  }
-
   render() {
-    const { employees, owners, pets } = this.state;
     return (
       <div className="wrapper">
-        <Nav />
-        <Home employees={employees} owners={owners} pets={pets} />
-        <StaffList employees={employees} />
-        <PetsList pets={pets} />
-        <Footer />
+        <Switch>
```

Comment:
Since the Nav and footer is always rendering you can have the component called outside the switch statement and won't have to add it to each route.

```
return (
  <div>
    <nav/>
    <switch>
       ...
    </switch>
    <footer/>
  </div>
)
```